js/tldjs.es6.js

```diff
@@ -0,0 +1,5 @@
+/*
+ * Require tldjs module
+ * This will be browserifyed by running grunt
+ */
+URLParser = require('tldjs');
```

Comment:
Can you wire this up as `tldjs` vs `URLParser`?

Reply:
Actually it would probably be most efficient long-term to also rewire `utils.parseURL` to `tldjs.parse` and update all usages of `parseURL` (there's only two).